src/app/style-panel/style-panel.component.ts

```diff
@@ -0,0 +1,127 @@
+import { Component, AfterViewInit, ViewChildren } from '@angular/core';
+import { NgForm } from '@angular/forms';
+import { Store } from '@ngrx/store';
+import { getFields, getGeneralStyle } from '../core/store/index';
+import { StyleServices } from '../shared/style.services';
+import { getStyle } from '../core/store/index';
+import {
+  ChangeStyleAction,
+  RemoveStyleAction,
+} from '../core/store/styles-fields/styleFields.actions';
+import { ChangeStyleGeneralAction } from '../core/store/styles-general/general-style.actions';
+import { valueDefault } from '../shared/value.sheets';
+import { FormArray, FormControl, FormGroup, Validators } from '@angular/forms';
+import { Observable, Observer } from 'rxjs';
+
+enum Attribute {
```

Comment:
Enum - E
Interfave - I
```suggestion
enum EAttribute {
```